server/views/pages/about.njk

```diff
@@ -77,6 +83,16 @@
 
             <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-3"/>
 
+            <h1 class="govuk-heading-m">Licences for a person both recalled and serving another sentence</h1>
```

Comment:
Not sure, but I think multiple `H1` on a page causes an accessibility error?

Reply:
Thanks - not noticed these, should be h2's I think. Will change now.